.eslintrc.json

```diff
@@ -4,7 +4,8 @@
   "parserOptions": { "ecmaVersion": 10 },
   "env": {
     "node": true,
-    "es6": true
+    "es6": true,
+    "mocha":true
```

Comment:
Adding this here tells ESLint that the Mocha globals are in-scope in all the files, which isn't true. You should make a new file at `tests/.eslintrc.json` that adds this env, so that it applies only to that directory.